terminal-management/shopify.extension.toml.liquid

```diff
@@ -3,6 +3,8 @@ name = "{{ name }}"
 type = "terminal_management"
 
 terminals_url = "https://api.paymentprovider.com/terminals"
-terminal_status_url = "https://api.paymentprovider.com/terminal/status"
+terminal_details_url = "https://api.paymentprovider.com/terminal/details"
 # This is the page on your website where merchants can go to diagnose problems.
-terminal_diagnostic_page_url = "https://paymentprovider.com/help"
+terminal_diagnostic_page_url = "https://api.paymentprovider.com/terminal/setup_help"
+terminal_encryption_settings_url = "https://api.paymentprovider.com/terminal/encryption_settings"
+provider_location_mappings_url = "https://api.example.com/location_mappings"
```

Comment:
Is the `provider` prefix necessary? All the urls point to the provider so `location_mappings_url` is probably sufficient.